### PR TITLE
Refactor: Catch and Log Error

### PR DESCRIPTION
**Pull request by haricnugraha**
This PR catch and log error, so user can easily understand what is wrong when error happen.

### CODE REVIEW

File: src/utils/looper.ts [hunk 1]
```diff
@@ -21,22 +18,27 @@ async function doProbes(config: Config) {
...
-      notifications: config.notifications,
-      url: item.request.url ?? '',
-    })
+      await sendAlerts({
+        validations: validatedResp,
+        notifications: config.notifications,
+        url: item.request.url ?? '',
+      })
+    } catch (error) {
+      console.error(error.message)
```
Comment (dennypradipta) [inline src/utils/looper.ts:33]: Now that you destructured the `log` from `console`, why don't you just do it like
`import { log, error} from 'console'`?

Is there any reasons?

Comment (haricnugraha) [reply src/utils/looper.ts:33]: eslint rule https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/catch-error-name.md

Comment (dennypradipta) [reply src/utils/looper.ts:33]: I have read the link provided, it is said that you could use descriptive name:
```
// Descriptive name is allowed
try {} catch (fsError) {}
```

But I think it's not a big deal, just a matter of preferences, anyway. I'll mark this as resolved.

**Review by dennypradipta — APPROVED** [review]

File: src/utils/looper.ts (hunk 1 above)
Comment (haricnugraha) [reply src/utils/looper.ts:33]: it's allowed if fsError variable is not used.

**Review by sapiderman — APPROVED** [review]

**Pull request merged**